connector_magento/models/magento_storeview/common.py

```diff
@@ -74,32 +74,35 @@ def import_sale_orders(self):
                 sale_binding_model = sale_binding_model.sudo(user)
 
             backend = storeview.sudo(user).backend_id
-            if storeview.import_orders_from_date:
-                from_string = fields.Datetime.from_string
-                from_date = from_string(storeview.import_orders_from_date)
-            else:
-                from_date = None
+            from_date = storeview.import_orders_from_date
+
+            # Apply the global order import delay
+            to_date = import_start_time - timedelta(
+                minutes=backend.order_import_delay)
+            if from_date and to_date < from_date:
+                continue
 
             delayable = sale_binding_model.with_delay(priority=1)
             filters = {
                 'magento_storeview_id': storeview.external_id,
-                'from_date': from_date,
-                'to_date': import_start_time,
+                'from_date': (fields.Datetime.to_string(from_date)
+                              if from_date else None),
```

Comment:
if `False` is also fine, you don't need this check: https://github.com/OCA/OCB/blob/12.0/odoo/fields.py#L1790

Reply:
Nooooo

Reply:
I don't understand my initial answer anymore, but the answer is that False is not a valid value for this key.